src/app/service/util.service.ts

```diff
@@ -8,4 +7,7 @@ export class UtilService {
 
   constructor() { }
 
+  hideLoader(): void {
+    document.getElementById("loader")!.style.display="none";
```

Comment:
can you use a subject to show or hide a loader instead? subject can be declared in this file